src/IO_Strings.f90

```diff
@@ -1726,4 +1726,37 @@ FUNCTION str_SDK_nDecimal(r,nDecimal) RESULT(string)
 
     ENDFUNCTION str_SDK_nDecimal
 !
+!-------------------------------------------------------------------------------
+!> @brief Returns a character array by repeating a provided sample
```

Comment:
If we keep this function (see below), need to fill in this banner more with `@param` and `@returns` lines

Reply:
Just getting rid of it. We can use the intrinsic